Makefile

```diff
@@ -52,6 +52,7 @@ FIREFOX_ANDROID_REPO = @mozilla-mobile/firefox-android
 FIREFOX_IOS_REPO     = @mozilla-mobile/firefox-ios
 FOCUS_IOS_REPO       = @mozilla-mobile/focus-ios
 MONITOR_REPO         = @mozilla/blurts-server
+VPN_REPO             = @mozilla-services/guardian-website
```

Comment:
This isn't needed. These were only used before manifest tool did fetches. All of these constants can be removed.

Reply:
done, removed all of them